.github/workflows/rust.yml

```diff
@@ -0,0 +1,99 @@
+name: Rust
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
```

Comment:
Id prefer this on every branch. I don't always work on master.

Reply:
Ok, I'll fix that.